### PR TITLE
Use containerd cli for CI

### PR DESCRIPTION
**Pull request by Random-Liu**
Related to https://github.com/containerd/containerd/pull/2131.

Add `cmd/test-containerd` which is a customized version of containerd for CI.
We added warning:
```golang
// NOTE(random-liu): This customized containerd is only for CI test
// of cri plugin. NEVER USE IT IN PRODUCTION!
```

With this PR, we can get rid of the script to automatically update containerd vendor.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -40,24 +40,25 @@ default: help
...
-	@echo " * 'install.deps'     - Install dependencies of cri-containerd (containerd, runc, cni) Note: BUILDTAGS defaults to 'seccomp apparmor' for runc build"
-	@echo " * 'uninstall'        - Remove installed binaries from system locations"
-	@echo " * 'version'          - Print current cri-containerd release version"
-	@echo " * 'update-vendor'    - Syncs containerd/vendor.conf -> vendor.conf and sorts vendor.conf"
+	@echo " * 'install'          		- Install binaries to system locations"
+	@echo " * 'binaries'         		- Build cri-containerd and ctrcri"
+	@echo " * 'static-binaries   		- Build static cri-containerd and ctrcri"
+	@echo " * 'release'          		- Build release tarball"
+	@echo " * 'push'             		- Push release tarball to GCS"
+	@echo " * 'test-containerd'  		- Build a customized containerd with CRI plugin for testing"
```
Comment (mikebrow) [inline Makefile]: /s/test-//

File: Makefile [hunk 2]
```diff
@@ -40,24 +40,25 @@ default: help
...
+	@echo " * 'install-test-containerd' 	- Install customized containerd to system location"
```
Comment (mikebrow) [inline Makefile]: /s/test-//

File: cmd/test-containerd/test_containerd.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
+/*
```
Comment (mikebrow) [inline cmd/test-containerd/test_containerd.go]: move / rename "cmd/test-containerd/test_containerd.go" to "test/containerd/containerd.go"

File: cmd/test-containerd/test_containerd.go [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+	_ "github.com/containerd/containerd/services/tasks"
+	_ "github.com/containerd/containerd/services/version"
+	_ "github.com/containerd/containerd/snapshots/overlay"
+	_ "github.com/containerd/cri-containerd"
+
+	"github.com/containerd/containerd/cmd/containerd/command"
+)
+
+// NOTE(random-liu): This customized containerd is only for CI test
+// of cri plugin. NEVER USE IT IN PRODUCTION!
```
Comment (mikebrow) [inline cmd/test-containerd/test_containerd.go]: /s/NEVER USE IT IN PRODUCTION!/It is NOT for distribution, use in production, etc./

File: Makefile (hunk 1 above)
Comment (Random-Liu) [reply Makefile]: I want to emphasize that this is not a real containerd, it's just for testing.

Comment (Random-Liu) [reply Makefile]: Can you come up with a better name? My naming is bad.

File: Makefile (hunk 2 above)
Comment (Random-Liu) [reply Makefile]: Done.

File: cmd/test-containerd/test_containerd.go (hunk 3 above)
Comment (Random-Liu) [reply cmd/test-containerd/test_containerd.go]: Done.

File: cmd/test-containerd/test_containerd.go (hunk 4 above)
Comment (Random-Liu) [reply cmd/test-containerd/test_containerd.go]: Done.

**Comment by Random-Liu** [comment]
@mikebrow Added the suffix in the version string. And also added warning log.

File: Makefile [hunk 5]
```diff
@@ -40,24 +40,25 @@ default: help
...
-	@echo " * 'uninstall'        - Remove installed binaries from system locations"
-	@echo " * 'version'          - Print current cri-containerd release version"
-	@echo " * 'update-vendor'    - Syncs containerd/vendor.conf -> vendor.conf and sorts vendor.conf"
+	@echo " * 'install'          		- Install binaries to system locations"
+	@echo " * 'binaries'         		- Build cri-containerd and ctrcri"
+	@echo " * 'static-binaries   		- Build static cri-containerd and ctrcri"
+	@echo " * 'release'          		- Build release tarball"
+	@echo " * 'push'             		- Push release tarball to GCS"
+	@echo " * 'containerd'  		- Build a customized containerd with CRI plugin for testing"
+	@echo " * 'install-containerd' 	- Install customized containerd to system location"
```
Comment (mikebrow) [inline Makefile]: nit tab..

Comment (Random-Liu) [reply Makefile]: Will do. :)

Comment (Random-Liu) [reply Makefile]: Done.

**Pull request merged**